CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## v0.23.0
+### October 7, 2025
+
+* Bump go version to 1.25.1 (#274)
+* Automated dependency upgrades (#253)
+* update acceptance test for PR #257 (#259)
+* [COMPLIANCE] Add Copyright and License Headers (#273)
+* init changie (#272)
+* VAULT-36974: Updating backport assistant workflow (#270)
```

Comment:
Since these changelogs are meant to be read by the users, I assume we want to scrub them of Jiras?

Reply:
done, thanks for catching that.